dev/com.ibm.ws.jdbc/resources/com/ibm/ws/rsadapter/resources/IBMDataStoreAdapterNLS.nlsprops

```diff
@@ -245,6 +245,11 @@ DSRA4004.onconnect.sql=DSRA4004E: Unable to execute onConnect SQL command \"{0}\
 DSRA4004.onconnect.sql.explanation=An error occurred while attempting to execute the specified SQL command.
 DSRA4004.onconnect.sql.useraction=Refer to the exception message to help determine the cause of the problem.
 
+#4008			TRANSACTION_NONE_UNSUPPORTED
+TRANSACTION_NONE_UNSUPPORTED=DSRA4008: The JDBC driver does not support an isolation level of TRANSACTION_NONE.  The isolation level was set to TRANSACTION_NONE because of the data source configuration.
+TRANSACTION_NONE_UNSUPPORTED.explanation= The JDBC driver does not support using an isolation level of TRANSACTION_NONE.  An attempt to set the isolation level to TRANSACTION_NONE was the result of the data source configuration.
```

Comment:
TRANSACTION_NONE_UNSUPPORTED.explanation= The JDBC driver does not support using an isolation level of TRANSACTION_NONE. **The data source configuration caused an attempt to set the isolation level to TRANSACTION_NONE.**

Reply:
Need to remove extra white space at beginning of message.  Otherwise, this seem a bit wordy and awkward. Can we just say,
"The TRANSACTION_NONE isolation level that is configured on the data source is not supported by the JDBC driver."
?

Reply:
"The TRANSACTION_NONE isolation level that is configured on the data source is not supported by the JDBC driver." Looks good to me.